### ISSUE TITLE
SegmentedPropertyTypeModifierCodeSequence (left/right flag) lost in round trip

### ISSUE THREAD

**Issue by pieper** [herrmannlab/highdicom#203]
Using [this public SEG dataset](https://www.dropbox.com/s/9eesbv22hsaj4sn/ABD_LYMPH_008_SEG.dcm?dl=0) I tried the following:

```
>>> seg = highdicom.seg.segread("/opt/data/idc/seg-performance/dicom-seg/full-body-seg/ABD_LYMPH_008 Dennis Bontempi/ABD_LYMPH_008_SEG.dcm")
>>> seg.save_as("/tmp/seg.dcm")
```

Comparing the dumps of the original (left) and the newly saved file (right) I notice that the left/right modifiers are missing as in the example below.  There are some other differences but I think they are inconsequential.

![image](https://user-images.githubusercontent.com/126077/198665921-ad09f360-1246-437c-afe5-9f1af0053139.png)

**Comment by CPBridge** [comment]
Yikes, thanks for reporting this @pieper . Definitely a high priority to fix!

**Comment by CPBridge** [comment]
Resolved with #205
